pkgs/by-name/em/ember-cli/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, mkYarnPackage
+, fetchFromGitHub
+, fetchYarnDeps
+}:
+
+let
+  pname = "ember-cli";
+  version = "5.3.0";
+in
+mkYarnPackage rec {
```

Comment:
no point in having the `let` expr if we're going to have `rec` anyway, I'd remove the `let` tbh

Reply:
alternatively, `src` can be moved into the `let` block, then we can remove `rec`.